engine/tutorials/usingdocker.md

```diff
@@ -90,29 +90,29 @@ the important stuff: running more containers. So far none of the
 containers you've run did anything particularly useful, so you can
 change that by running an example web application in Docker.
 
-For our web application we're going to run a Python Flask application.
+For the web application you're going to run a Python Flask application.
 Start with a `docker run` command.
 
     $ docker run -d -P training/webapp python app.py
 
 Review what the command did. You've specified two flags: `-d` and
 `-P`. You've already seen the `-d` flag which tells Docker to run the
 container in the background. The `-P` flag is new and tells Docker to
-map any required network ports inside our container to our host. This
-lets us view our web application.
+map any required network ports inside the container to your host. This
+lets you view the web application.
 
 You've specified an image: `training/webapp`. This image is a
-pre-built image you've created that contains a simple Python Flask web
-application.
+pre-built image that contains a simple Python Flask web application.
 
-Lastly, you've specified a command for our container to run: `python app.py`. This launches our web application.
+Lastly, you've specified a command for the container to run:
```

Comment:
Can you consider rewriting to avoid the "you've specified"? Of course, if you try rewriting as in the previous comment, you may be able to shorten this a lot or not need it at all.